x/signal/README.md

```diff
@@ -27,18 +27,18 @@ See [types/msgs.go](./types/msgs.go) for the message types.
 
 ```shell
 celestia-appd query upgrade tally
```

Comment:
I assume the query command is also moving from the upgrade module to the signal module

```suggestion
celestia-appd query signal tally
```